R/extract_lyrics.R

```diff
@@ -9,37 +9,36 @@
 #' @examples
 #' extract_lyrics( "22", "Taylor Swift")
 extract_lyrics <- function(song_title, artist) {
-
     if ( stringr::str_squish(song_title) == "" | stringr::str_squish(artist) == 0){
         stop("Empty input")
-    }
+   }
 
     if ( class(song_title) != "character" | class(artist) != "character"){
-        stop("Invalid column type, song title and artist have to be strings")
+       stop("Invalid column type, song title and artist have to be strings")
     }
 
-    lyrics = ""
-    url = paste0(
-        "https://genius.com/",
-        stringr::str_replace_all(artist, " ", "-"),
-        "-",
-        stringr::str_replace_all(song_title, " ", "-"),
-        "-lyrics"
-    )
+     lyrics = ""
+     url = paste0(
+         "https://genius.com/",
+         stringr::str_replace_all(artist, " ", "-"),
+         "-",
+         stringr::str_replace_all(song_title, " ", "-"),
+         "-lyrics"
+     )
 
-    tryCatch({
-    Scraped_webpage <- rvest::read_html(url)
-    lyrics_html <- rvest::html_nodes(Scraped_webpage, xpath = '//*[@id="lyrics-root"]')
-    stripped_htmltext <- rvest::html_text(lyrics_html)
-    closeAllConnections()
-    },
-    error=function(cond) {
-        stop("Song not found")
-    })
+     tryCatch({
+     Scraped_webpage <- rvest::read_html(url)
+     lyrics_html <- rvest::html_nodes(Scraped_webpage, xpath = '//*[@id="lyrics-root"]')
+     stripped_htmltext <- rvest::html_text(lyrics_html)
+     closeAllConnections()
+     },
+     error=function(cond) {
+         stop("Song not found")
+     })
 
-    lyrics <- gsub("\\[[^][]*]"," ",stripped_htmltext)
-    lyrics <- gsub("([a-z])([A-Z])", "\\1 \\2", lyrics)
-    lyrics <- gsub(" Lyrics ", " ", lyrics)
-    lyrics
-}
+     lyrics <- gsub("\\[[^][]*]"," ",stripped_htmltext)
+     lyrics <- gsub("([a-z])([A-Z])", "\\1 \\2", lyrics)
+     lyrics <- gsub(" Lyrics ", " ", lyrics)
+     lyrics
+ }
```

Comment:
I am not sure why a single space has been added to these lines. Could you please remove them to original shape?

Reply:
Valid point, the same has been rectified